README.md

```diff
@@ -0,0 +1,38 @@
+# [Archivematica Storage Service](https://www.archivematica.org/)
+
+By [Artefactual](https://www.artefactual.com/)
+
+
+The Archivematica Storage Service is part of the Archivematica project.
+Archivematica is a web- and standards-based, open-source application which allows your institution to preserve long-term access to trustworthy, authentic and reliable digital content.
+The Storage Service is responsible for moving files to and from Archivematica for processing and long-term storage.
+For more information, please see the [Archivematica documentation](https://www.archivematica.org/en/docs/).
+
+You are free to copy, modify, and distribute Archivematica with attribution under the terms of the AGPL license.
+See the [LICENSE](LICENSE) file for details.
+
+
+## Resources
+
+* [Website](https://www.archivematica.org/): User and administrator documentation
+* [User Google Group](https://groups.google.com/forum/#!forum/archivematica): Forum/mailing list for user questions
+* [Wiki](https://www.archivematica.org/wiki/Development): Developer-facing documentation
+* [Technical Google Group](https://groups.google.com/forum/#!forum/archivematica-tech): Forum/mailing list for technical questions about development, setup, administration, etc.
+* [Paid support](https://www.artefactual.com/services/): Paid support, hosting, training, consulting and software development contracts from Artefactual
+
+
+## Contributing
+
+Thank you for your interest!
+For more details, please see the [Archivematica contributing guidelines](https://github.com/artefactual/archivematica/blob/qa/1.x/CONTRIBUTING.md).
```

Comment:
I think there should be a CONTRIBUTING.md in the repo, so Github points to it as you mentioned.  I also think linking to a qa branch is not ideal.